pkg/kstatus/status/status_augment_test.go

```diff
@@ -132,7 +132,6 @@ func TestAugmentConditions(t *testing.T) {
 	}
 
 	for tn, tc := range testCases {
-		tc := tc
```

Comment:
I know what you are talking about. This is no longer necessary since Go 1.22. I myself am not used to this yet so wasn't very comfortable making this change, but, hey, they've fixed the problem! See https://go.dev/blog/loopvar-preview.